.travis.yml

```diff
@@ -1,155 +0,0 @@
-# do not edit this file manually, instead run the generate:travis-yml console command
```

Comment:
If we won't use travis anymore, you can also remove the submodule `tests/travis` (if you don't use any files from there directly)